qa/rpc-tests/p2p-segwit.py

```diff
@@ -908,7 +907,7 @@ def test_witness_tx_relay_before_segwit_activation(self):
         # a witness transaction ought not result in a getdata.
         try:
             self.test_node.announce_tx_and_wait_for_getdata(tx, timeout=2)
-            print("Error: duplicate tx getdata!")
+            self.log.error("Error: duplicate tx getdata!")
             assert(False)
```

Comment:
You don't need the assert(False) when you pass any AssertionError below. Though maybe a fix for an unrelated follow up pr.